osu.Game/Screens/Play/Player.cs

```diff
@@ -385,14 +388,16 @@ private void updateBackgroundElements()
             var storyboardVisible = showStoryboard && beatmap.Storyboard.HasDrawable;
 
             storyboardContainer.FadeColour(new Color4(opacity, opacity, opacity, 1), 800);
-            storyboardContainer.FadeTo(storyboardVisible && opacity > 0 ? 1 : 0);
+            storyboardContainer.FadeTo(storyboardVisible && opacity > 0 ? 1 : 0, 800, Easing.OutQuint);
 
             Background?.FadeTo(!storyboardVisible || beatmap.Background == null ? opacity : 0, 800, Easing.OutQuint);
+            (Background as BackgroundScreenBeatmap)?.BlurTo(blur, 800, Easing.OutQuint);
         }
 
         private void fadeOut()
         {
-            dimLevel.ValueChanged -= dimLevel_ValueChanged;
+            dimLevel.ValueChanged -= backgroundLevel_ValueChanged;

```

Comment:
these aren't needed as we are using a local bindable

Reply:
i just followed what was done since 7c3ce7e830c9783c89ac6c8fafb943a831e5b6fb, however if this isn't necessary anymore i'd see no problem with removing either